bring the demul emulator to batocera (under wine)

dmanlfc opened the pull request:
Notes:
For running Sega - Naomi 2, Hikaru, Gaelco & Cave CV1000 games
Requires Wine - (x86_64 only)

nadenislamarre commented:
is it ok to merge for v33 ?

nadenislamarre commented:
shutil.copytree("/usr/demul", emupath) is not really nice. can we do without it ?

nadenislamarre commented:
please avoid to put .po/.pot updates in such pr (keep them in a distinct pr) because it is very subject to conflicts.

nadenislamarre commented:
for me, this pr is ok for validation if you validate it is for v33

Hew-ux commented:
Please rename:
TURN ON VSYNC > VSYNC
ASPECT RATIO > GAME ASPECT RATIO
for both consistency and to save the translators from extra, redundant work!

dmanlfc commented:
@nadenislamarre - yes good to go for v33
- updated to master just now
- removed translations
- shutil.copytree("/usr/demul", emupath) is necessary unfortunately - closed source emulator.

@Hew-ux - done.
